File: week1/mario.c
#include <stdio.h>
#include <cs50.h>

// A program that prompts user to insert a number & prints out that many question marks

int main(void)
{
    // Get positive integer for width
    int n;
    do
    {
        n = get_int("Width: ");
    }
    while (n < 1);
    
    // Print out that many question marks 
    for (int i = 0; i < n; i++)
    {
        printf("?");
    }
    printf("\n");
}

File: week6/mario.py
# A program that builds a grid of bricks inspired by mario

from cs50 import get_int

def main():
    n = get_int("How many #? ")
    if n > 0:
        print("Building mario grid of bricks...")
        for i in range(n):
            for j in range (n):
                print("#", end="")
            print()
    else:
        print("Enter a positive integer.")
        main()

main()